packages/serverless-orchestration/package.json

```diff
@@ -4,17 +4,15 @@
   "description": "Serverless orchestration scripts to run a parallel instances of scripts or bots",
   "dependencies": {
     "@uma/common": "^2.38.0",
-    "@uma/financial-templates-lib": "^2.37.0",
+    "@uma/logger": "^1.3.1",
     "web3": "^1.6.0"
   },
   "devDependencies": {
```

Comment:
fwiw, this set of `devDependencies` seems quite long relative to the functionalities we actually have in serverless-orchestration. I think it's worth a 2nd pass to see what else we can trim out.